include/SugarFolders/SugarFolders.php

```diff
@@ -510,71 +614,72 @@ public function getCountUnread($folderId)
 
     /**
      * Convenience method, pass a SugarBean and User to this to add anything to a given folder
+     *
+     * @param string $bean
+     * @param User   $user
+     * @return boolean
      */
     public function addBean($bean, $user = null)
```

Comment:
You could remove unused $user parameter as it does not needed and passed from nowhere..